src/Build/Instance/TaskRegistry.cs

```diff
@@ -1361,7 +1358,9 @@ private bool GetTaskFactory(TargetLoggingContext targetLoggingContext, ElementLo
                                 }
                                 finally
                                 {
+#if FEATURE_APPDOMAIN
```

Comment:
No the `taskFactoryLoggingHost` is still needed but the `MarkAsInactive()` method is not compiled into the .NET Core MSBuild.